spec/requests/resource_evaluations_controller_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe ResourceEvaluationsController, type: :request do
+  describe 'POST /create' do
+    let(:user) { create(:user) }
+    let(:resource) { create(:resource, user:) }
+    let(:params) do
+      { 'resource_id': resource.id, 'resource_evaluations':
+        { 'evaluation': 3 } }
+    end
+
+    def perform
+      post resource_resource_evaluations_path(params)
+    end
+
+    before do
+      sign_in user
+    end
+
+    context 'when trying to create an existing evaluation' do
+      before do
+        create(:resource_evaluation, user:, resource:, evaluation: 4)
+      end
+
+      it 'updates the evaluation value' do
```

Comment:
it 'doesn't update the evaluation value'

Reply:
No, it tests that it does udpate it.
Before the test the evaluation was set in 4, the test changes it to 3.